Cargo.toml

```diff
@@ -627,6 +627,11 @@ branch = "oxide/omicron"
 [patch.crates-io.omicron-workspace-hack]
 path = "workspace-hack"
 
+# Pulls in https://github.com/njaremko/samael/pull/41
+[patch.crates-io.samael]
+git = "https://github.com/oxidecomputer/samael"
+branch = "oxide/omicron"
```

Comment:
should we be pointing this at a rev instead of a branch, so that the build is reproducible from this commit forever?

Reply:
That's not part of this commit -- it's a merge artifact. [current main](https://github.com/oxidecomputer/omicron/blob/fedd8c87ccbf17af6251433a122652cf0962464e/Cargo.toml#L616-L619) has it.

Reply:
(But also -- doesn't `Cargo.lock` ensure reproducibility?)